frontend/src/utils/validate-template.ts

```diff
@@ -45,6 +51,39 @@ export const validateEmailTemplate = (
   }
 };
 
+export const validateSubmittedEmailTemplate = (
+  template: Template | undefined
+): SubmittedEmailTemplate | undefined => {
+  try {
+    return $SubmittedEmailTemplate.parse(template);
+  } catch (error) {
+    logger.error(error);
+    return undefined;
+  }
+};
+
+export const validateSubmittedSMSTemplate = (
+  template: Template | undefined
+): SubmittedSMSTemplate | undefined => {
+  try {
+    return $SubmittedSMSTemplate.parse(template);
+  } catch (error) {
+    logger.error(error);
+    return undefined;
+  }
+};
+
+export const validateSubmittedNHSAppTemplate = (
+  template: Template | undefined
+): SubmittedNHSAppTemplate | undefined => {
+  try {
```

Comment:
this is just `zodValidate($SubmittedNHSAppTemplate, template)`

Reply:
Yes, I'd spotted that we are using a mix of this and wrapper functions. I went with the wrapper, but happy to change if there's a reason not to?

Reply:
i would have gone with `zodValidate` as it's less code to maintain but not important, happy as is